db-init.sql

```diff
@@ -0,0 +1,166 @@
+-- PersonCategories
+	-- all the different types a person can be
+
+CREATE TABLE PersonCategories (
+	personCategoryId SERIAL PRIMARY KEY NOT NULL,
+	personCategoryName VARCHAR(255) NOT NULL,
+);
+
+	CREATE UNIQUE INDEX personCategoryName_lower ON PersonCategories(lower(personCategoryName));
+
+	INSERT INTO PersonCategories VALUES (0, 'farmers');
+	INSERT INTO PersonCategories VALUES (1, 'traders');
+	INSERT INTO PersonCategories VALUES (2, 'admins');
+	INSERT INTO PersonCategories VALUES (3, 'monitor');
```

Comment:
all the other ones are plural, should monitor be plural? also how do you feel about naming these "viewers" instead? "monitor" might be misleading because it could also be a verb (ie in terms of checking something)

Reply:
he said he likes monitors 👍